code-reviews/recipes/CSharp.md

```diff
@@ -6,15 +6,80 @@
 
 ## Linting
 
-[CSE](../../CSE.md) projects should check code style with the [StyleCop Analyzers for the .NET Compiler Platform](https://github.com/DotNetAnalyzers/StyleCopAnalyzers). The minimum rules set teams should adopt is the [Managed Recommended Rules](https://msdn.microsoft.com/en-us/library/dd264893.aspx) rule set.
+We strongly believe that consistent style increases readability and maintainability of a code base. Hence we are recommending analyzers / linters to enforce consistency and style rules.
 
-Reference: [Setting up code analysis for a project](https://docs.microsoft.com/en-us/visualstudio/code-quality/code-analysis-for-managed-code-overview).
+### Project Setup
+
+We recommend using a common setup for your solution that you can refer to in all the projects that are part of the solution. Create a common.props file that contains the defaults for all of your projects:
+
+```xml
+ <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+  <ItemGroup Condition="Exists('$(MSBuildThisFileDirectory)../.editorconfig')" >
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)../.editorconfig" />
+  </ItemGroup>
+```
+
+You can then reference the common.props in your other project files to ensure a consistent setup.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="..\common.props" />
+</Project>
+```
+
+The [.editorconfig](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2019) allows for configuration and overrides of rules. You can have an .editorconfig file on project level to customize rules for different projects (test projects for example).
+
+Details about the configuration of different rules can be found [here](https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019).
```

Comment:
Nit: It is usually recommended to avoid linking on 'here' - this could just say [Details about the configuration of different rules](https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019)

I'll let you decide if you want to change or not

Reply:
Thanks! Are those rules somewhere published? I'd like to read up on them.